gotests.go

```diff
@@ -16,6 +16,13 @@ import (
 	"github.com/cweill/gotests/internal/output"
 )
 
+var (
+	cmpImport = &models.Import{
+		Name: "",
+		Path: `"github.com/google/go-cmp/cmp"`,
+	}
+)
```

Comment:
minor: is there any other data it should be grouped in the future?

```suggestion
var cmpImport = &models.Import{
	Name: "",
	Path: `"github.com/google/go-cmp/cmp"`,
}
```